hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -579,7 +601,6 @@ public boolean rollback(final String commitTime) throws HoodieRollbackException
         return true;
     }
 
-
     private void rollback(List<String> commits) {
```

Comment:
right things to do here is to move rollback and clean into the `HoodieTable` interface..

Reply:
okay my bad. you have done that below, for rollback..